lib/screens/gl_search.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:git_touch/models/theme.dart';
+import 'package:git_touch/scaffolds/common.dart';
+import 'package:git_touch/utils/utils.dart';
+import 'package:git_touch/widgets/issue_item.dart';
+import 'package:git_touch/widgets/loading.dart';
+import 'package:git_touch/widgets/user_item.dart';
+import 'package:primer/primer.dart';
+import 'package:provider/provider.dart';
+import 'package:git_touch/models/auth.dart';
+import 'package:git_touch/widgets/repository_item.dart';
+import 'package:timeago/timeago.dart' as timeago;
+import 'package:git_touch/models/gitlab.dart';
+
+class GlSearchScreen extends StatefulWidget {
+  @override
+  _GlSearchScreenState createState() => _GlSearchScreenState();
+}
+
+class _GlSearchScreenState extends State<GlSearchScreen> {
+  int _activeTab = 0;
+  bool _loading = false;
+  List<List> _payloads = [[], []];
```

Comment:
Great work!

I've tested it at my local machine and there seems some errors:

```
[VERBOSE-2:ui_dart_state.cc(157)] Unhandled Exception: type '_InternalLinkedHashMap<String, dynamic>' is not a subtype of type 'Iterable<dynamic>'
```

Perhaps caused by the implicit `dynamic` here?

Reply:
Perhaps [tuple](https://pub.dev/packages/tuple) could solve this? Or simply split them into several variables

Reply:
I used separate variables. Also, while running on the latest version of flutter, I had to update `flutter_svg` in `pubspec.yaml` to a newer version as there were build errors. Should I open a separate pr for that or add a commit here itself?

Reply:
Yeah, I faced this issue too. I think Both should be fine